workflow/tests/util.rb

```diff
@@ -247,6 +250,9 @@ def _verify_outputs(rundir, hpxml_path, results, hpxml, unit_multiplier)
     if hpxml_path.include? 'base-hvac-multiple.xml'
       next if message.include? 'Reached a minimum of 1 borehole; setting bore depth to the minimum'
     end
+    if hpxml_path.include? 'base-hvac-furnace-gas-central-ac-var-speed-max-power-ratio-schedule.xml'
+      next if message.include? 'Maximum power ratio schedule is only supported for variable speed systems'
+    end
```

Comment:
I don't understand, why does this file generate a warning? It has a variable speed system.

Reply:
Probably it's not the best way to implement this, but if either heating/cooling or both is not variable speed, the schedule doesn't apply for that period of time, in this case, heating is not applied. Maybe we just ignore the heating by default w/o throwing any warning?

Reply:
I would think you should check whether the **system** (i.e., heating _or_ cooling) has a multispeed coil and give a warning if the system doesn't have one. Don't independently check and warn about heating and cooling w/o multispeed coils.

Reply:
Sounds fair to me. Addressed.